core/modules/parsers/parseutils.js

```diff
@@ -385,4 +385,35 @@ exports.parseAttribute = function(source,pos) {
 	return node;
 };
 
+/*
+Given a parsed attribute node, serialize it back into its source string representation.
+*/
+exports.serializeAttribute = function(node) {
+	if(!node || typeof node !== 'object' || !node.name || !node.type) {
+			return null;
+	}
+
+	var attributeString = node.name;
+
```

Comment:
TW doesn't use blank lines for readability in code, preferring comments instead

Reply:
Updated with formatting.